### PR TITLE
 #167866494 improve database functionality

### PR DESCRIPTION
**Pull request by NiyongaboEric**
### What does this PR do?
Chore implementation increases the security of the application.  the test response has a common display flow and mock files are separated from the test files. 

#### Expected
- Code should be Modulize
-  use of ES6
- Create response flow
- Create Mockfile for static files
- hide admin credentials

### How should this be manually tested
- Test Trips routes
>  URL/api/v1/trips

### What are the relevant pivotal tracker stories? 
[#167866494](https://www.pivotaltracker.com/story/show/167866494)

### Screenshoots

HEROKU Admin login
admin-keys-hidden

JWT read key from Heroku
jwt-admin

PGADMIN Create key
test-db

Travis recognize Admin key
test-moched-passed

### CODE REVIEW

**Comment by Kasulejoseph** [comment]
Change the PR title to show the business value it's adding to the application

**Comment by NiyongaboEric** [comment]
@Kasulejoseph I changed

File: server/controllers/UserController.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ class UserController {
         id: userInfo.rows[0].id, email: userInfo.rows[0].email, is_admin: userInfo.rows[0].is_admin,
       }, process.env.SECRET_KEY, { expiresIn: '1D' }, (err, token) => {
         data.token = token;
-        return res.status(200).json({ status: 200, message: 'welcome to way-farer', data });
+        return Customize.CommonResponse(res, res, 'User Created Successfully', data, 200);
```
Comment (Kasulejoseph) [inline server/controllers/UserController.js]: Inconsistency in your code eg `Customize` and `customize`,  PascalCase vs camelCase

File: server/helpers/customize.js [hunk 2]
```diff
@@ -15,6 +15,30 @@ class Customize {
     res.status(status).json({ status, error: msg });
   }
 
+  /**
```
Comment (Kasulejoseph) [inline server/helpers/customize.js]: Stick on OOP principles even in this file.

File: server/controllers/UserController.js (hunk 1 above)
Comment (NiyongaboEric) [reply server/controllers/UserController.js]: i noticed too

File: server/helpers/customize.js (hunk 2 above)
Comment (NiyongaboEric) [reply server/helpers/customize.js]: let fix both

File: server/controllers/UserController.js (hunk 1 above)
Comment (NiyongaboEric) [reply server/controllers/UserController.js]: Fixed

File: server/helpers/customize.js (hunk 2 above)
Comment (NiyongaboEric) [reply server/helpers/customize.js]: fixed

File: server/controllers/trip/TripController.js [hunk 3]
```diff
@@ -37,11 +37,7 @@ class TripController {
     const data = { ...result.rows[0] };
     data.trip_date = new Date(data.trip_date).toDateString();
     data.created_at = new Date(data.created_at).toDateString();
-    return res.status(201).json({
-      status: 201,
-      message: 'Trip created successfully',
-      data,
-    });
+    return customize.CommonResponse(res, res, 'Trip created successfully', data, 201);
```
Comment (Kasulejoseph) [inline server/controllers/trip/TripController.js]: Why are you using customize with `c` here yet in other files you use Customize with `C`

**Pull request closed without merging**